src/libraries/System.Runtime.Serialization.Formatters/tests/System.Runtime.Serialization.Formatters.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-FreeBSD;$(NetCoreAppCurrent)-illumos;$(NetCoreAppCurrent)-Solaris;$(NetCoreAppCurrent)-Linux;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS;net48</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="BinaryFormatterEventSourceTests.cs" Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'" />
```

Comment:
Is it going to do the right thing? The list of TargetFrameworks above only has `$(NetCoreAppCurrent)-Something` items.

Reply:
That is fine, when the outer build dispatches the inner builds it strips out the platform from the configuration when defining the property `TargetFramework`, which is why you will see that in other projects we have conditions like `'$(TargetFramework)' == '$(NetCoreAppCurrent)' and '$(TargetsWindows)' == 'true'` which will end up applying only on `$(NetCoreAppCurrent)-Windows_NT` configuration.

In this particular case, this condition is making it so that this file gets compiled everywhere but the `net48` configuration

Reply:
Nope, I verified that this works from VS but is broken from the command line. (I wrote a dummy unit test that always fails, and xunit didn't execute it.) Will update the .csproj accordingly.